setup.py

```diff
@@ -1,10 +1,11 @@
 #!/usr/bin/env python3
 
-from setuptools import setup
+from setuptools import find_packages, setup
 
 
 setup(
     author='ckolos',
+    author_email='ckolos@mozilla.com',
```

Comment:
setuptools gets sad if you include an `author` but not `author_email`.

Reply:
makes sense.